UndertaleModTool/Editors/UndertaleEmbeddedAudioEditor.xaml.cs

```diff
@@ -119,11 +119,11 @@ private void Play_Click(object sender, RoutedEventArgs e)
                         waveOut.Init(oggReader);
                         waveOut.Play();
                     } else
-                        MessageBox.Show("Failed to play audio!\r\nNot a WAV or OGG.", "Audio failure", MessageBoxButton.OK, MessageBoxImage.Warning);
+                        MessageBox.Show(Application.Current.MainWindow as MainWindow, "Failed to play audio!\r\nNot a WAV or OGG.", "Audio failure", MessageBoxButton.OK, MessageBoxImage.Warning);
                 } catch (Exception ex)
                 {
                     waveOut = null;
-                    MessageBox.Show("Failed to play audio!\r\n" + ex.Message, "Audio failure", MessageBoxButton.OK, MessageBoxImage.Warning);
+                    MessageBox.Show(Application.Current.MainWindow as MainWindow, "Failed to play audio!\r\n" + ex.Message, "Audio failure", MessageBoxButton.OK, MessageBoxImage.Warning);
```

Comment:
Can you create a property for this, so it doesn't have to get copypasted to every catch?